Best practices 1.1 (#1271)
OctopusDeploy/docs: docs/getting-started/best-practices/installation-guidelines.md

3. Restart the server and wait for it to come back online.
4. Remove the drain mode from the node.

> ## Create a Single Production Instance

Reviewer: ```suggestion
## Create a single Production instance
```

One question we get asked a lot is "should we have a single instance to deploy to all environments or have an Octopus Deploy instance per environment?"  Unless there is a business requirement, our recommendation is to have a single instance to deploy to all environments and use Octopus Deploy's RBAC controls to control permissions.  We recommend this to avoid the maintenance overhead involved with having an instance per environment.  

Of the users who opt for an instance per environment, we see them have an instance for **development** and **test** environments with another instance for **staging** and **production** environments.  

> If you go down that route you would need a process to:

Reviewer: ```suggestion
If you chose this instance configuration, you would need a process to:
```

- Clone all the library variables and project variables, and notify you when a new scoped variable is added.
- Sync the deployment and runbook processes, but skip over steps assigned to **development** and **test**.
- Update any user step templates to the latest version.
- Ensure the same lifecycle names exist on both instances but not have the same phases.
- Copy releases but not deployments.
- Clone all the project channels.
- And more.

> Using the Octopus Deploy API, all of that is possible, however it will require dilligence and maintenance on your part.  Unless there is a specific business requirement, such as security team requirements or regulatory requirements, we don't recommend taking that on.

Reviewer: ```suggestion
Using the Octopus Deploy API, all of that is possible; however, it will require diligence and maintenance on your part.  Unless there is a specific business requirement, such as security team requirements or regulatory requirements, we don't recommend taking that on.
```

> ## Further Reading

Reviewer: ```suggestion
## Further reading
```
